cogboard-webapp/src/components/FormMessages.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled/macro';
+
+
+const FormMessages = ({className, messages, ...others}) => {
+
+  if (messages) {
```

Comment:
Don't use if else in render. Do the early return the null if no messages and return the regular component.

Reply:
done